tools/build/wheel_linux_before_all.sh

```diff
@@ -1,25 +1,27 @@
-#!/bin/bash
+#!/bin/sh
 
 # Configure the libraries needed to build wheel packages on linux.
 # This script is designed to be used by cibuildwheel as CIBW_BEFORE_ALL_LINUX
 
-set -euo pipefail
-set -x
+set -xe
 
-dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+. /etc/os-release
 
-# Install libpq
-# Note that the pgdg doesn't have an aarch64 repository so wheels are build
-# with the libpq packaged with Debian 9, which is 9.6.
-if [[ ! "$AUDITWHEEL_ARCH" = "aarch64" ]]; then
-    source /etc/os-release
-    echo "deb http://apt.postgresql.org/pub/repos/apt ${VERSION_CODENAME}-pgdg main" \
-        > /etc/apt/sources.list.d/pgdg.list
-    # TODO: on 2021-09-30 curl fails with
-    # curl: (60) SSL certificate problem: certificate has expired
-    # Test again later if -k can be removed.
-    curl -sk https://www.postgresql.org/media/keys/ACCC4CF8.asc \
-        | apt-key add -
-fi
-apt-get update
-apt-get -y install libpq-dev
+# Install PostgreSQL development files.
+case "$ID" in
+    alpine)
+        apk add --no-cache postgresql-dev
```

Comment:
What libpq version do you get?

Reply:
12.8.

```console
# apk info postgresql-dev
postgresql-dev-12.8-r0 description:
A sophisticated object-relational DBMS (development files)

postgresql-dev-12.8-r0 webpage:
https://www.postgresql.org/

postgresql-dev-12.8-r0 installed size:
9117696
```

Reply:
Let's go with it for the moment. Getting v14 would be better in the near future (3.1 should ship with some features depending on it, which would be unavailable in these packages otherwise).

Reply:
Unfortunately even the latest Alpine version (3.14) only has PostgreSQL 13. So even if [pypa/manylinux](https://github.com/pypa/manylinux) updates the base image from Alpine 3.12 to 3.14, it won't get us to PostgreSQL 14.

I'm not super well versed in Alpine's release cycle. But I suspect we'd have to wait for Alpine 3.16 for PostgreSQL 14 (or newer).

Reply:
Did at bit of experimenting, and it seems like it's possible to install packages from Alpine's 'edge' repository (presumably their upcoming 3.15 release). And it includes `postgresql-dev-14.0-r6`.

So it worst case scenario, that could probably be hacked together for a psycopg 3.1 release. But until then it's probably best to stay with the 'main' repositoy since using 'edge' is likely a bit brittle.

```console
$ apk add --repository https://dl-cdn.alpinelinux.org/alpine/edge/main 'postgresql-dev>=14'
...
(21/21) Installing postgresql-dev (14.0-r6)
Executing busybox-1.31.1-r20.trigger
OK: 322 MiB in 33 packages
```

Reply:
I think it's ok to wait for when those package will become stable :)